packages/cli/src/cli/config/action.import.ts

```diff
@@ -73,16 +82,29 @@ export class CommandImport extends CommandLineAction {
       if (!res.ok) throw new Error('Basemaps is unhealthy');
     }
 
+    if (backup) {
+      await fsa.writeJson(backup, this.backupConfig.toJson());
+    }
+
     if (commit !== true) logger.info('DryRun:Done');
   }
 
   update(config: BaseConfig, commit: boolean): void {
     const promise = Q(async (): Promise<boolean> => {
       const updater = new Updater(config, commit);
+      const db = updater.getDB();
+
       const hasChanges = await updater.reconcile();
-      if (hasChanges) this.invalidations.push(updater.invalidatePath());
+      if (hasChanges) {
+        this.invalidations.push(updater.invalidatePath());
+        const oldData = await db.get(config.id);
```

Comment:
this should already be fetched insider `updater.reconcile()`? could we have `updater.dbConfig` or similar to not fetch it multiple times?

Reply:
Fixed.